[2022-04-14 10:10:58,845] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 2022-04-13T00:00:00+00:00 [queued]>
[2022-04-14 10:10:58,937] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 2022-04-13T00:00:00+00:00 [queued]>
[2022-04-14 10:10:58,948] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 10:10:58,948] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-14 10:10:58,949] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 10:10:59,019] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): processing_tasks.task_2> on 2022-04-13T00:00:00+00:00
[2022-04-14 10:10:59,028] {standard_task_runner.py:52} INFO - Started process 23339 to run task
[2022-04-14 10:10:59,043] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'processing_tasks.task_2', '2022-04-13T00:00:00+00:00', '--job-id', '60', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpzk5766i5', '--error-file', '/tmp/tmp7i895euo']
[2022-04-14 10:10:59,055] {standard_task_runner.py:77} INFO - Job 60: Subtask processing_tasks.task_2
[2022-04-14 10:10:59,321] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_2 2022-04-13T00:00:00+00:00 [running]> on host airflowvm
[2022-04-14 10:10:59,588] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-13T00:00:00+00:00
[2022-04-14 10:10:59,589] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-04-14 10:10:59,590] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-14 10:10:59,614] {subprocess.py:75} INFO - Output:
[2022-04-14 10:11:02,634] {subprocess.py:83} INFO - Command exited with return code 0
[2022-04-14 10:11:02,787] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=processing_tasks.task_2, execution_date=20220413T000000, start_date=20220414T101058, end_date=20220414T101102
[2022-04-14 10:11:03,003] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-14 10:11:03,043] {local_task_job.py:151} INFO - Task exited with return code 0
